src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/package.json

```diff
@@ -22,31 +22,35 @@
     "@angular/platform-server": "8.2.12",
     "@angular/router": "8.2.12",
     "@nguniversal/module-map-ngfactory-loader": "8.1.1",
-    "bootstrap": "^4.3.1",
-    "core-js": "^3.3.3",
+    "bootstrap": "^4.6.0",
```

Comment:
@captainsafia can you have a look at this merge? Also, did you perform any verification on this?